packages/ts/react-grid/src/autogrid.tsx

```diff
@@ -182,7 +185,9 @@ export function AutoGrid<TItem>({
   useEffect(() => {
     // Sets the data provider, should be done only once
     const grid = ref.current!;
-    grid.dataProvider = createDataProvider(grid, service, dataProviderFilter);
+    setTimeout(() => {
```

Comment:
Could add a comment to explain the timeout.

Reply:
Added